src/store/metadata/oxigraph/store.py

```diff
@@ -145,11 +145,38 @@ def get_topics(self) -> Optional[Dict]:  # pragma: no cover
         """
         raise NotImplementedError
 
-    def get_topic(self) -> Optional[Dict]:  # pragma: no cover
+    def get_topic(self, topic_id: str) -> Optional[Dict]:  # pragma: no cover
```

Comment:
you can remove the # pragma: no cover please. It's just there so we didnt feedback of a lack of test coverage for a method that wasn't implemented.